lib/chat_web/live/main_live/index.html.heex

```diff
@@ -433,7 +433,12 @@
               </ul>
             </div>
 
-            <Layout.Uploader.uploader config={@uploads.file} uploads={@uploads_metadata} />
+            <Layout.Uploader.uploader
+              config={@uploads.file}
+              pub_key={@peer && @peer.pub_key}
```

Comment:
`assigns[:peer][:pub_key]` does the same. Just less hack-ish way

Reply:
That was my first approach, but it requires `Card` to implement Access protocol.
```
** (UndefinedFunctionError) function Chat.Card.fetch/2 is undefined (Chat.Card does not implement the Access behaviour. If you are using get_in/put_in/update_in, you can specify the field to be accessed using Access.key!/1)
    (chat 0.1.0) Chat.Card.fetch(#Chat.Card<name: "Mac", hash: "f04b0226f4bfb79363d0efc5a65671a08c34ea1eae5ec8374caf8e9c6b74f4e3", ...>, :pub_key)
```

Reply:
https://hex.pm/packages/struct_access
and `use StructAccess` in Card